client/src/components/Workflow/Run/WorkflowRunFormSimple.vue

```diff
@@ -5,6 +5,7 @@
 <script>
 import Form from "mvc/form/form-view";
 import { invokeWorkflow } from "./services";
+import isWorkflowInput from "components/Workflow/constants";
```

Comment:
```suggestion
import { isWorkflowInput } from "components/Workflow/constants";
```

Reply:
Not sure I grok `constants` either -- what else goes in here?  Might be more `util`?

Reply:
Yeah, I initially just had the array in there, but thought a function might be nicer. I can make this util, or, considering we're not re-suing this yet anywhere, I could also add this straight to WorkflowRunFormSimple.vue. Seems like we'd need this for invocation step displaying as well, but I can move this when we actually need it.